docker-build-components-example.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+: '
+Make a copy of this file and rename to docker-build-components.sh
+as the renamed file is in .gitignore. This will avoid commiting the file
+with certificate information.
+'
+echo "Exporting environment variables"
+#Put your party key here (OpenTest):
+export MHS_SECRET_PARTY_KEY=""
+
+#Put your endpoint certificate here (OpenTest):
+MHS_SECRET_CLIENT_CERT=""
+
+#Put your endpoint private key here (OpenTest):
+MHS_SECRET_CLIENT_KEY=""
+
+#Put your Endpoint issuing subCA certificate and your Root CA certificate here (OpenTest)
+MHS_SECRET_CA_CERTS=""
+
+#Stops containers, builds images and starts containers
+echo "Stopping running containers"
+docker-compose -f docker-compose.yml -f docker-compose.lb.override.yml stop;
+
+echo "Build and starting containers"
+docker-compose -f docker-compose.yml -f docker-compose.lb.override.yml up -d --build
```

Comment:
Personal preference really, but I tend to run "attached" or without the -d flag. Sometimes the logs in the console are useful.

Reply:
Makes sense, i only do this so a user does not try to carry on using the shell and accidently exits from their start up. Thoughts on this as a use case?